andela/travel_tool_front
# # 162007599 Validate File Uploads

#### What does this PR do?
This PR enforces validations based on file type and size when a user attempts to upload a new file

#### Description of Task to be completed?

- Display message indicating allowed file size

- Display message indicating allowed file type

- Return error message if user adds incorrect file type/size

#### How should this be manually tested?

- Clone this repository: `git clone https://github.com/andela/travel_tool_front.git`

- Checkout to this branch ``

- Install dependencies using `yarn install`

- Start the application's backend and frontend using `make start`

- Navigate to the `Documents` view in the application and `click on add file`

- The add file modal should display messages on file size and type allowed

- An attempt to upload incorrect file type/size should display an error message


#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[162007599](https://www.pivotaltracker.com/story/show/162007599)

#### Screenshots (if appropriate)


#### Questions:
N/A
